plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -774,6 +775,13 @@ export class SessionRecordingIngester {
                     })
             }
         )
+        const errors = results.filter((r) => !!r.error).map((r) => r.error)
+        if (errors.length) {
+            status.error('🌶️', 'blob_ingester_consumer - failed to flush sessions', { errors })
+            throw new Error(
```

Comment:
so if any flushes error we'll throw here (which currently isn't caught anywhere so will kill the pod

it does mean we might have flushed some sessions and then not commit the offsets

but probably safer to die and pick up again here 🦐

Reply:
i believe these are rare _but_ we could roll this out without a throw and then check the logs to confirm